resources/stsci.edu/schemas/general_sersic2d-1.0.0.yaml

```
@@ -0,0 +1,64 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/general_sersic2d-1.0.0"
+title: >
+  Two dimensional generalized Sersic surface brightness profile.
+
+description: >
+  Two dimensional Sersic surface brightness profile over a generalized ellipse.
+
+examples:
+  -
+    - $I(x, y)=I(r)=I(x, y)=I(r)=10.0\exp\left\{-b_n\left[\left(\frac{(\sqrt{(x-0.5)^2 + (y-1.5)^2})}{1.0}\right)^{(1/4)}-1\right]\right\}$ where $b_n$ is defined such that $r_e$ contains half the total luminosity (can be solved for numerically).
```

```suggestion
    - $I(x, y) = I_{e} \exp\left\{-b_{n} \left[\left(\frac{r(x, y)}{r_{e}}\right)^{(1/n)}-1\right]\right\}$ where $I_e$ is the amplitude, $b_n$ is defined such that $r_e$ (r_eff) contains half the total luminosity (can be solved for numerically).
```
I attempted to update this formula and description to more closely match the [astropy docs](https://docs.astropy.org/en/latest/api/astropy.modeling.functional_models.GeneralSersic2D.html#generalsersic2d). I think there's room for further improvement to describe `isophate/c`, etc.